browser/scripts/node.js

```diff
@@ -368,66 +368,82 @@ Node.prototype.update_connections = function() {
 	return this.inputs.length + this.outputs.length
 }
 
+/**
+ * set connection UI flow state to off recursively
+ */
+function cascadeFlowOff(conn) {
+	conn.ui.flow = false
+	if (conn.src_node.inputs.length)
```

Comment:
can probably early out after the first connection, if the next one is already off